flameshot_example.sh

```diff
@@ -2,7 +2,7 @@
 IMAGEPATH="$HOME/Pictures/" # Where to store screenshots before they're deleted
 IMAGENAME="ass" # Not really important, tells Flameshot what file to send and delete
 KEY="" # Your ass upload token
-DOMAIN="" # Your upload domain
+DOMAIN="" # Your upload domain (without http(s))
 
```

Comment:
Can you please revert this change? Some users do use HTTPS with Flameshot

Reply:
Actually hold on I may have misunderstood

Reply:
@Metacinnabar Can you actually *change* that comment to be:

```bash
DOMAIN="" # Your upload domain (without http:// or https://)

Reply:
Done!